tests/cases/unittests/commandLineParsing.ts

```diff
@@ -216,10 +216,24 @@ namespace ts {
                         file: undefined,
                         start: undefined,
                         length: undefined,
-                    }, {
-                        messageText: "Argument for '--lib' option must be:  'es5', 'es6', 'es2015', 'es7', 'es2016', 'es2017', 'dom', 'webworker', 'scripthost', 'es2015.core', 'es2015.collection', 'es2015.generator', 'es2015.iterable', 'es2015.promise', 'es2015.proxy', 'es2015.reflect', 'es2015.symbol', 'es2015.symbol.wellknown', 'es2016.array.include', 'es2017.object', 'es2017.sharedmemory'",

```

Comment:
Shouldn't all 3 cases be errors? I don't think `lib: []` is valid, so somebody along the line should throw an error.